xonsh/lexer.py

```diff
@@ -42,7 +42,7 @@
     '&=': 'AMPERSANDEQUAL', '^=': 'XOREQUAL', '|=': 'PIPEEQUAL',
     '//=': 'DOUBLEDIVEQUAL',
     # extra xonsh operators
-    '?': 'QUESTION', '??': 'DOUBLE_QUESTION', '@$': 'ATDOLLAR', '!': 'BANG',
+    '?': 'QUESTION', '??': 'DOUBLE_QUESTION', '@$': 'ATDOLLAR',
```

Comment:
Ahh, yeah, BANG was removed because of it was no longer used.  It showed up as a PLY warning, so I thought I would clean it up.